### PR TITLE
Add tests for updating/deleting shipping

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I want to add tests for updating and deleting shipping methods and zones

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://stable.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5019 Add tests for shipping Method](https://app.clickup.com/t/2549495/SALEOR-5019)

File: cypress/fixtures/urlList.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ export const urlList = {
   pageTypes: "page-types/",
   permissionsGroups: "permission-groups/",
   plugins: "plugins/",
+  priceRete: "price/",
```
Comment (Cloud11PL) [inline cypress/fixtures/urlList.js:21]: Rete?

**Review by Cloud11PL — APPROVED** [review]

File: cypress/fixtures/urlList.js (hunk 1 above)
Comment (karola312) [reply cypress/fixtures/urlList.js:21]: PriceRate, I fixed it

**Review by eCzerniak — CHANGES_REQUESTED** [review]

File: cypress/fixtures/urlList.js [hunk 2]
```diff
@@ -63,6 +64,9 @@ export const userDetailsUrl = userId => `${urlList.staffMembers}${userId}`;
 export const weightRateUrl = (shippingZoneId, weightRateId) =>
   `${urlList.shippingMethods}${shippingZoneId}/${urlList.weightRete}${weightRateId}`;
 
+export const priceRateUrl = (shippingZoneId, priceRateId) =>
+  `${urlList.shippingMethods}${shippingZoneId}/${urlList.priceRete}${priceRateId}`;
```
Comment (eCzerniak) [inline cypress/fixtures/urlList.js:68]:
```suggestion
  `${urlList.shippingMethods}${shippingZoneId}/${urlList.priceRate}${priceRateId}`;
```

**Review by eCzerniak — APPROVED** [review]
Please change this one line and it will be OK.

**Pull request merged**